code/game/turfs/simulated/walls_reinforced.dm

```diff
@@ -163,7 +163,7 @@
 					if( d_state == 4 && user.loc == T && user.get_active_hand() == W )
 						src.d_state = 5
 						src.icon_state = "r_wall-5"
-						user << "<span class='notice'>You remove the bolts anchoring the support rods.</span>"
+						user << "<span class='notice'>You remove the bolts which you thought were anchoring the support rods.</span>"

```

Comment:
Sry I suppose I should've explained that without asking. First loosening the anchoring points of something, then just outright welding them off seemed funny to me so I made it seem like you (or anyone for that matter) don't really know how to deconstruct the rwall and are starting to get frustrated with it. ...Is it too far fetched?